README.md

```diff
@@ -75,7 +75,7 @@ in which case you may not even need the asset pipeline. This is mostly relevant
 * Ruby 2.2+
 * Rails 4.2+
 * Node.js 6.4.0+
-* Yarn 0.20.1+
+* Yarn 0.20.1+ or NPM 4.2.0+
```

Comment:
Are they compatible with all versions of node? In webpacker node > 6.4 is required. I noticed later npm versions dropped support for `node ~> 0.12`